src/virtualkeyboardlayout.h

```diff
@@ -35,6 +35,17 @@ class KeyboardLayout {
         }
     }
 
+    ~KeyboardLayout() {
+        FCITX_KEYBOARD_LAYOUT() << "~KeyboardLayout()";
+        keys_.clear();
```

Comment:
It seems that we need also call `delete` for each elements, since they are allocated by `new`.

Reply:
And they aren't smart pointer.